dockstore-integration-testing/src/test/java/io/dockstore/client/cli/OrganizationIT.java

```diff
@@ -1314,6 +1312,25 @@ public void testBasicCollections() {
             fail("Was able to reject an approved collection");
         }
 
+        // version 8 and 9 belong to entryId 2
+        long versionId = 9L;
+
+        // Add tool and specific version to collection
+        organizationsApi.addEntryToCollection(organization.getId(), collectionId, entryId, versionId);
+        organizationsApi.addEntryToCollection(organization.getId(), collectionId, entryId, null);
+
+        // There should now be two entries for collection with ID 1 (one with version, one without), 3 entries in total
+        collectionById = organizationsApi.getCollectionById(organizationID, collectionId);
+        assertEquals(3, collectionById.getEntries().size());
```

Comment:
Codacy found an issue: ['3' is a magic number.](https://app.codacy.com/gh/dockstore/dockstore/pullRequest?prid=6286359)